docs-site/src/markdown-pages/index.md

```diff
@@ -0,0 +1,113 @@
+---
+key: index
+---
+
+## Overview
+
+EOP is an initiative led by Greater Wellington Regional Council building on the
+experience of the Environment Canterbury Water Data programme to build a
+solution which will allow all regional councils to maximise the value they get
+from their environmental monitoring efforts and provide clear, consistent, and
+user-centric information to users of environmental information.
+
+Regional councils have many systems for collecting and managing environmental
+data. These systems are often siloed, legacy, and not integrated. This makes
+integrating the data to derive information a manual task of combining the data
+from multiple systems, which is slow, error-prone, and often this is done in a
+way that isn't consistent within a single council or across councils. This
+effort to integrate the data limits the type of analysis that people will even
+try to achieve with the data. This leads to the value of the data not being
+fully realised.
+
+Much work has been done in councils by people without a software or data
+engineering background on the best efforts' basis. This has resulted in
+solutions that deliver lots of value but lack the maintainability, scalability
+and user-centric approach that would come from a software engineering team.
```

Comment:
Trying to say there are a lot of "Not great quality" solutions out there ... but not sure its needed or worded well.

Really using it as a line to introduce why bringing in a team that follows "Modern Development Practices" is an important step

Reply:
Sounding good. And good idea to frame-up the value of "Modern Development Practices" in the last para.

I've taken a stab at word-smithing, but I'm not precious about it so feel free to use or ignore. It's mainly just breaking down long sentences and removing some duplication. I also bought back the statement explaining what it is (an end-to-end platform for capture...) in the opening sentence as that seemed like a good thing to start with?

> EOP is an initiative led by Greater Wellington Regional Council to establish an end-to-end platform for capture, management and reporting of environmental information. The goal is to allow all regional councils to maximise the value they get from their environmental monitoring efforts, and provide clear, consistent and user-centric information to end-users if this data. It’s built based on the experiences from the Environment Canterbury Water Data programme 
> 
> Regional councils have many systems for collecting and managing environmental data. These systems are often siloed and based on legacy technology. Integrating this data can be a slow, manual and and error prone process that is not consistent within or across councils. These challenges to limits the type of analysis that is feasible, meaning the value of the data is not fully realised.
> 
> Much work has been done in councils by people without a software or data engineering background on a best efforts' and as-needed basis. This has resulted in solutions that deliver lots of value but lack the maintainability, scalability and user-centric approach that would come from a dedicated software engineering team.